jnpr/openclos/testcase.py

```diff
@@ -0,0 +1,123 @@
+import re
```

Comment:
What is the purpose of this testcase, if it for unit test move this file under .../tests/unit